lib/okta/service.rb

```diff
@@ -31,6 +32,24 @@ def app(app_id)
       end
     end
 
+    def get_apps(url)
+      with_monitoring do
+        get_url_with_token(url)
```

Comment:
You may be able to combine the get_apps and get_auth_servers methods. They look pretty similar to me and just pass in a url as a param. Just a suggestion.

Reply:
get_apps is a piece of dead code, since the apps are now being stored in records instead of grabbing them externally from Okta. I'll remove the get_apps function